src/main/kotlin/no/nav/syfo/kafka/KafkaModule.kt

```diff
@@ -65,4 +66,6 @@ fun launchKafkaModule(
         applicationState = applicationState,
         kafkaEnvironment = environment.kafka,
     )
+    ArbeidsuforhetVurderingConsumer(personoversiktStatusService = personoversiktStatusService)
```

Comment:
Litt usikker på om jeg synes denne måten å gjøre det på er en forbedring i forhold til hvordan det er gjort ovenfor, men det kan vi kanskje ta en diskusjon på: hvilken struktur gjør koden enklest å forstå, og legger til rette for å lage tester?